demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -0,0 +1,289 @@
+    output = np.zeros(target_shape, dtype=np.uint8)
+    scale = min(output_resolution[1] / h, output_resolution[0] / w)
+    input_img = cv2.resize(input_img, None, fx=scale, fy=scale)
+    dw = output_resolution[0] - input_img.shape[1]
+    dh = output_resolution[1] - input_img.shape[0]
+    x0 = dw // 2
+    x1 = dw // 2 if dw % 2 == 0 else dw // 2 + 1
+    y0 = dh // 2
+    y1 = dh // 2 if dh % 2 == 0 else dh // 2 + 1
+    output[y0: output.shape[0] - y1, x0: output.shape[1] - x1, :] = input_img
+    return output
+
+
+def render_results(frame, objects, output_resolution, target_bgr, person_id, blur_bgr=False, show_with_original_frame=False):
+    if target_bgr is None:
+        target_bgr = np.full(frame.shape, [155, 255, 120], dtype=np.uint8)
+    else:
+        target_bgr = cv2.resize(target_bgr, (frame.shape[1], frame.shape[0]))
+    classes, masks = objects[1], objects[3]
+    # Choose masks only for person class
+    valid_inds = classes == person_id
+    masks = [mask for mask, is_valid in zip(masks, valid_inds) if is_valid]
+    if not len(masks):
+        output = frame
+    else:
+        composed_mask = masks[0]
+        for i in range(1, len(masks)):
+            composed_mask = np.logical_or(composed_mask, masks[i])
+        # Smooth contours of the predicted mask
+        composed_mask = cv2.medianBlur(composed_mask.astype(np.uint8), 11)
+        composed_mask = np.repeat(np.expand_dims(composed_mask, axis=-1), 3, axis=2)
+        if target_bgr is not None and blur_bgr:
+            target_bgr = cv2.blur(cv2.resize(target_bgr, (target_bgr.shape[1], target_bgr.shape[0])), (7, 7))
+        output = np.where(composed_mask == 1, frame, target_bgr)
+    if show_with_original_frame:
+        output = cv2.hconcat([frame, output])
+    h, w = output.shape[:2]
+    if (w, h) != tuple(output_resolution):
+        output = fit_to_window(output, output_resolution)
+    return output
+
+
+def main():
+    args = build_argparser().parse_args()
+
+    cap = open_images_capture(args.input, args.loop)
+
+    target_bgr = open_images_capture(args.target_bgr, loop=True) if args.target_bgr else None
+
+    if args.adapter == 'openvino':
+        plugin_config = get_user_config(args.device, args.num_streams, args.num_threads)
+        model_adapter = OpenvinoAdapter(create_core(), args.model, device=args.device, plugin_config=plugin_config,
+                                        max_num_requests=args.num_infer_requests)
+    elif args.adapter == 'remote':
+        log.info('Reading model {}'.format(args.model))
+        serving_config = {"address": "localhost", "port": 9000}
+        model_adapter = RemoteAdapter(args.model, serving_config)
+
+    labels = ['__background__', 'person'] if args.labels is None else args.labels
+
+    model = get_instance_segmentation_model(model_adapter, prob_threshold=args.prob_threshold,
+                                            labels=labels, keep_aspect_ratio=args.keep_aspect_ratio)
```

Comment:
```suggestion
    configuration = {
        'prob_threshold': args.prob_threshlod,
        'labels': labels,
        'keep_aspect_ratio': args.keep_aspect_ratio,
    }
    model = get_instance_segmentation_model(model_adapter, configuration)
```
Please, modify the demo and the model wrappers according to the latest version of our model API:
1) Model wrapper constructor accepts the configuration dictionary with parameters values
2) Model wrapper implements `parameters()` method and has a `__model__` attribute

You might refer to object_detection_demo and segmentation_demo as examples.

Reply:
Fixed.